modules/ROOT/pages/uploadcare.adoc

```diff
@@ -0,0 +1,206 @@
+= {pluginname} plugin
+:navtitle: Image Optimizer Powered by Uploadcare
+:description: The Image Optimizer Powered by Uploadcare plugin allows you to optimize images in your content.
+:description_short: 
+:plugincode: uploadcare
+:pluginname: Image Optimizer Powered by Uploadcare
+:keywords: plugin, {plugincode}, image, align, transform, alt text, caption, adjust, filter
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-7.6v.adoc[]
+
+== Overview
+
+The **Image Optimizer** plugin leverages a suite of powerful features for hosting, serving and edit images like responsive image delivery, automatic image file format selection, automatic compression and non-destructive image transformations and adjustments, all powered by link:https://uploadcare.com[Uploadcare].
+
+== Key benefits
+
+* Improve page loading speed and save on bandwidth by serving images in the optimal size, format, and compression based on the visitor's browser and device.
+* Ditch the image editor and adjust images right inside {productname} with non-destructive transformations and adjustments. Change your mind any time!
+* Stay safe and compliant with secure uploads, built in malware protection, and unsafe content detection
+* Enterprise-ready scalable cloud storage provided by Uploadcare, delivered through its lightning-fast global CDN, and compliant with SOC2 and GDPR standards.
+
+== Interactive example
+
+liveDemo::{plugincode}[]
+
+== Basic setup
+
+To add the {pluginname} plugin to the editor, add `{plugincode}` to the `plugins` option in the editor configuration.
+
+For example:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'uploadcare',
+  toolbar: 'uploadcare',
+  uploadcare_public_key: '<your-public-key>',
+  // uploadcare_signed_upload_auth_provider: (_publicKey) => Promise.resolve({
+  //   signature: 'sig',
+  //   expire: 123
+  // }), // Recommended option for secure uploads
+});
+----
+
+[NOTE]
+The {pluginname} plugin overrides the xref:quickbars.adoc[Quickbar] plugin. To ensure a better user experience, avoid using xref:quickbars.adoc[Quickbar] when using the {pluginname} plugin.
+
+== Image Operations
+
+Below is an overview of the features of the {pluginname} plugin includes for image optimization:
+
+[cols="1,1,^1,3",options="header"]
+|===
+| Feature | Operation | Icon | Description
+
+| Align
+|
+| image:icons/align-left.svg[align-left.svg]
+| Provides alignment options for the image or element.
+
+|
+| Align Left
+| image:icons/align-left.svg[align-left.svg]
+| Aligns the image to the left.
+
+|
+| Align Centre
+| image:icons/align-center.svg[align-center.svg]
+| Centers the image.
+
+|
+| Align Right
+| image:icons/align-right.svg[align-right.svg]
+| Aligns the image to the right.
+
+| Transform
+|
+| image:icons/transform-image.svg[transform-image.svg]
+| Allows users to apply transformations to the image:
+
+|
+| Rotate Counterclockwise
```

Comment:
This was last minute renamed to `Rotate left` both terms seems fine the clockwise/counterclockwise is more precise but left and right might be more user friendly. I guess we can still keep the counterclockwise description then it hard to miss what it's doing from the docs.

Reply:
Updated in last commit.